package.json

```diff
@@ -46,6 +46,7 @@
     "react-native": "^0.67.4",
     "react-native-webview": "^11.17.2",
     "typescript": "^4.5.4",
-    "vite": "^2.8.0"
+    "vite": "5.0",
```

Comment:
We should use the `^` to fix this version to its major version but let the patch and minor versions update automatically. You can read more about the `^` operator [here](https://betterstack.com/community/questions/difference-between-tilde-and-caret-in-package-json/), but in short it helps us maintain our packages at least updated with security patches and stuff like that automatically when we update some other package. I would say we can trust Vite to follow semver strictly, so using `^` should be OK.

Reply:
I'll add it, thanks!